common/fileview2/fvwugen.cpp

```diff
@@ -182,7 +182,7 @@ IHqlExpression * PositionTransformer::createTransformed(IHqlExpression * _expr)
                 {
                     IHqlExpression * newTarget = createField(child->queryId(), child->getType(), LINK(child), insertedAttr.getLink());
                     fields.append(*newTarget);
-                    assigns.append(*createValue(no_assign, makeVoidType(), newTarget, createSelectExpr(createValue(no_left), LINK(newTarget))));
+                    assigns.append(*createValue(no_assign, makeVoidType(), newTarget, createSelectExpr(createValue(no_left, makeNullType()), LINK(newTarget)))); // MORE: This probably doesn't work
```

Comment:
Should probably comment why this doesn't work.

Reply:
ok

Reply:
Updated comment - but note, code is deprecated and should really be deleted.